runtime/v2/manager.go

```diff
@@ -139,14 +144,166 @@ func (m *MM) ValidateGenesis(genesisData map[string]json.RawMessage) error {
 	return nil
 }
 
-// InitGenesis performs init genesis functionality for modules.
-func (m *MM) InitGenesis() {
-	panic("implement me")
+func (m *MM) InitGenesis(ctx context.Context, genesisData map[string]json.RawMessage) (*abci.ResponseInitChain, error) {
```

Comment:
We shouldn't need to return an abci type, just returning a slice a appmodule validator updates should do it

Reply:
got it